app/config/locale/translations/sk.json

```diff
@@ -0,0 +1,232 @@
+{
+    "settings.inspire": "\"Umenie múdrosti je umenie vedieť, čo prehliadnuť.\"",
+    "settings.locale": "sk",
+    "settings.direction": "ltr",
+    "emails.sender": "%s Tím",
+    "emails.verification.subject": "Overenie účtu",
+    "emails.verification.hello": "Ahoj {{name}}",
+    "emails.verification.body": "Použi tento link pre overenie svojej emailovej adresy.",
+    "emails.verification.footer": "Ak si nepožiadal o overenie tejto adresy, môžeš túto správu ignorovať.",
+    "emails.verification.thanks": "Ďakujeme.",
+    "emails.verification.signature": "{{project}} tím",
+    "emails.magicSession.subject": "Prihlásenie",
+    "emails.magicSession.hello": "Ahoj,",
+    "emails.magicSession.body": "Použi tento link pre prihlásenie.",
+    "emails.magicSession.footer": "Ak si nepožiadal o prihlásenie cez email, túto správu môžeš ignorovať.",
+    "emails.magicSession.thanks": "Ďakujeme",
+    "emails.magicSession.signature": "{{project}} tím",
+    "emails.recovery.subject": "Obnovenie hesla",
+    "emails.recovery.hello": "Ahoj {{name}}",
+    "emails.recovery.body": "Použi tento link pre obnovenie svojho {{project}} hesla.",
+    "emails.recovery.footer": "Ak si nepožiadal o obnovu svojho hesla, túto správu môžeš ignorovať.",
+    "emails.recovery.thanks": "Ďakujeme",
+    "emails.recovery.signature": "{{project}} tím",
+    "emails.invitation.subject": "Pozvánka do %s Tímu v %s",
+    "emails.invitation.hello": "Ahoj",
+    "emails.invitation.body": "Tento email ti bol zaslaný, pretože {{owner}} ťa chce pozvať aby si sa stal členom {{team}} tímu v {{project}}.",
```

Comment:
I would go with:
```
... zaslaný, pretože {{owner}} ťa pozval, aby si sa stal členom {{team}} tímu v projekte {{project}}.",
```

Reply:
Yeah, agreed, that sounds better. Should I make a new pull request for this?